java/code/internal/src/com/redhat/rhn/internal/doclet/DocBookDoclet.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2012 SUSE LLC
+ * Copyright (c) 2020 SUSE LLC
```

Comment:
```suggestion
 * Copyright (c) 2012--2020 SUSE LLC
```

Reply:
I have recently been told in PR #2063 to avoid ranges :)

Reply:
If Legal told you so, this is ok. I would like to see the mail on galaxy-devel so we can change the process for all the files. What I know is, that in spacewalk they bumped the year for all files every year.

Reply:
Ciaran said we should use ranges, fixed